ember-cli-build.js

```diff
@@ -5,7 +5,9 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 
 module.exports = function(defaults) {
   let app = new EmberAddon(defaults, {
-    // Add options here
+    'ember-cli-babel': {
+      includePolyfill: true
```

Comment:
Lets use https://github.com/machty/ember-maybe-import-regenerator (as a dev dep) instead of adding the whole polyfill

Reply:
Awesome, thanks for the tip. Added and removed unnecessary polyfill.